build/files.c

```diff
@@ -2659,24 +2659,285 @@ static int checkFiles(const char *buildRoot, StringBuf fileList)
     return rc;
 }
 
+static rpmTag copyTagsForDebug[] = {
+    /* see copyTagsDuringParse */
+    RPMTAG_EPOCH,
+    RPMTAG_VERSION,
+    RPMTAG_RELEASE,
+    RPMTAG_LICENSE,
+    RPMTAG_PACKAGER,
+    RPMTAG_DISTRIBUTION,
+    RPMTAG_DISTURL,
+    RPMTAG_VENDOR,
+    RPMTAG_ICON,
+    RPMTAG_URL,
+    RPMTAG_VCS,
+    RPMTAG_CHANGELOGTIME,
+    RPMTAG_CHANGELOGNAME,
+    RPMTAG_CHANGELOGTEXT,
+    RPMTAG_PREFIXES,
+    RPMTAG_DISTTAG,
+    RPMTAG_BUGURL,
+    0
+};
+
```

Comment:
Hmm, AFAICS that only differs by RPMTAG_GROUP from copyTagsDuringParse. I can see wanting to use specific group for debuginfo, but  why not reuse copyTagsDuringParse and then just delete RPMTAG_GROUP? Would be less duplication - unless there are other reasons and differences that aren't obvious from the outset.